src/backend/base/langflow/utils/util_strings.py

```diff
@@ -28,3 +30,14 @@ def truncate_long_strings(data, max_length=None):
                 truncate_long_strings(item, max_length)
 
     return data
+
+
+def to_pythonic_variable_name(name):
+    """Converts a given string into a Pythonic variable name in snake_case."""
+    # Remove any characters that are not alphanumeric or spaces
+    name = re.sub(r"[^\w\s]", "", name)
+    # Replace spaces or other delimiters with underscores
+    name = re.sub(r"[\s]+", "_", name)
+    # Convert to lowercase
+    name = name.lower()
+    return name
```

Comment:
We should probably check if it starts with a number as well. Although, this function is not being used. Do you have plans for it?

Reply:
@ogabrielluiz yes I intend to use this and expect others to do so as well. This is particularly useful in Tools, which need a name that is variable-friendly. This will then enable a tool to get the name and description from values users have set in the UI (on the _vertex).